doc/internal/metadocumentation.txt

```diff
@@ -116,20 +116,19 @@ Pylint documentation is generated using pylintrc file: ``Theano/doc/pylintrc``
 The nightly build/tests process
 ---------------------------------------
 
-The user ``lisa`` runs a cronjob on the computer ``ceylon``,  this
-happens nightly. (To have the crontab executed, the ``lisa`` user must
-be logged into ``ceylon``, Fred leaves a shell open for that.)
+We use the Jenkins software to run daily buildbots for Theano, libgpuarray and
+the Deep Learning Tutorials. Jenkins downloads/updates the repos and then runs their test
+scripts. Those scripts test the projects under various condition.
+Jenkins also run some tests in 32 bit Python 2.7 and Python 3.5 for Theano.
```

Comment:
I see on this page that we use Python 3.4, not 3.5: http://darjeeling.iro.umontreal.ca:8080/job/Theano_buildbot_python3/lastBuild/consoleFull

Also, on that page, we see some reference to Python 3.3. They should be updated to 3.4. Keep jenkins with Python 3.4 for now.